README.md

```diff
@@ -2,7 +2,6 @@
 
 [![GitHub](https://img.shields.io/github/license/a-scie/jump)](LICENSE)
 [![Github Actions CI (x86_64 Linux / MacOS / Windows)](https://github.com/a-scie/jump/actions/workflows/ci.yml/badge.svg)](https://github.com/a-scie/jump/actions/workflows/ci.yml)
```

Comment:
Does the x86_64 reference here need an update too? (I guess this applies to the other two PRs I just reviewed too...)

Reply:
Aha - yeah, sloppy. Thanks.